### PR TITLE
doc: develop: tools: CLion: Add update note

### PR DESCRIPTION
**Pull request by olegznv**
CLion now offers a [more native approach](https://www.jetbrains.com/help/clion/zephyr.html) to working with Zephyr projects. We would like to inform users about this.

### CODE REVIEW

File: doc/develop/tools/clion.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 CLion
 #####
 
+> **Updated:** This guide describes how to set up, build, and debug Zephyr’s sample application in CLion, using the IDE’s CMake integration. This approach is no longer optimal.
+>
+> CLion now features [native Zephyr West integration](https://www.jetbrains.com/help/clion/zephyr.html), which provides an easier and more intuitive way to open, build, and run/debug Zephyr projects. This guide will be updated soon, but it’s still valid if you prefer to use CMake.
+
```
Comment (kartben) [inline doc/develop/tools/clion.rst:9]: we use ReStructured Text, not markdown :)
you prob want something like the following (make sure to add the actual hyperling associated with "native Zephyr West integration" at the bottom of the file.

```suggestion
.. note::

   This guide describes how to set up, build, and debug Zephyr's sample application in CLion, using
   the IDE's CMake integration. This approach is no longer optimal.

   CLion now features `native Zephyr West integration`_ which provides an easier and more intuitive
   way to open, build, and run/debug Zephyr projects. This guide will be updated soon, but is still
   valid if you prefer to use CMake.
```

**Review by kartben — CHANGES_REQUESTED** [review]

File: doc/develop/tools/clion.rst (hunk 1 above)
Comment (olegznv) [reply doc/develop/tools/clion.rst:9]: Oh, got it :) Thanks for the help @kartben!

**Comment by kartben** [comment]
@olegznv will you be able to look into squashing the commits? I understand you've made the contribution through the GitHub web UI, so you'll have to work from a local clone to do this. Let me know if I can help

**Comment by olegznv** [comment]
@kartben I'll try to do this. Thanks!

**Comment by olegznv** [comment]
@kartben I've squashed commits. Please check

**Review by kartben — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Pull request merged**